.circleci/config.yml

```diff
@@ -123,18 +123,64 @@ workflows:
           rails_version: "5.1.7"
           requires:
             - bundle
+      - test:
+          name: "rails5-1"
+          ruby_version: "2.5.8"
+          requires:
+            - build
+  ruby2-5rails5-2:
+    jobs:
+      - bundle:
+          ruby_version: "2.5.8"
+          rails_version: "5.2.4.3"
+      - build:
+          ruby_version: "2.5.8"
+          rails_version: "5.2.4.3"
+          requires:
+            - bundle
+      - lint:
+          ruby_version: "2.5.8"
+          requires:
+            - bundle
       - test:
           name: "ruby2-5-8"
           ruby_version: "2.5.8"
           requires:
             - build
-            - lint
+  ruby2-6rails5-2:
+    jobs:
+      - bundle:
+          ruby_version: "2.6.6"
+          rails_version: "5.2.4.3"
+      - build:
+          ruby_version: "2.6.6"
+          rails_version: "5.2.4.3"
+          requires:
+            - bundle
+      - lint:
```

Comment:
i think we can do without the multiple lints?

historically, we've only run this job on the oldest target ruby version, since that's the one we need to pass syntax support for. it looks like you could avoid some test setup issues by dropping these builds.

Reply:
Thanks, works for me. I am a total circleci noob. I've gotten the rails5.2 workflows all passing, but I'm having trouble with the (newish) ruby2.7.1 workflow. If you've got an idea of how to configure it, I'll gladly surrender the branch back to you. Then we might actually be all green (!)

Reply:
Thanks, works for me. I am a total circleci noob. I've gotten the rails5.2 workflows all passing, but I'm having trouble with the (newish) ruby2.7.1 workflow. If you've got an idea of how to configure it, I'll gladly surrender the branch back to you. Then we might actually be all green (!)

Reply:
Actually circleci is complaining `Job 'ruby2-6-6' requires 'lint', which is the name of 0 other jobs in workflow 'ruby2-6rails5-2'`. Hmmm.

Reply:
I put the lints back and circleci is running again. I'm going to step back from this and actually read some circleci documentation instead of throwing things at the wall hoping it sticks

Reply:
I put the lints back and circleci seem happy with that, not sure why. I'm going to step back from this and actually read some circleci documentation. Not sure why it seems to want the lints or why ruby2.7 is failing...

Reply:
Yeah, please do, I'd appreciate it.